app/src/main/java/com/microsoft/device/samples/dualscreenexperience/presentation/order/OrderFragment.kt

```diff
@@ -144,6 +145,12 @@ class OrderFragment : Fragment() {
         super.onResume()
         setupConfirmationObservers()
         changeToolbarTitle()
+        Log.i("HISTORY_TEST", "OrderFragment: onResume")
```

Comment:
Are we using proguard to remove logs in production? (Or any other custom mechanism or library)? All these logs will be shown also in production if we don't handle it and I guess we don't want that.

Reply:
that's a great point, I'll remove them for now from this branch and then if we decide we want them merged in, we can set up the proguard rules to remove the logs in production!